crates/re_viewer/src/log_db_hub.rs

```diff
@@ -0,0 +1,151 @@
+use re_data_store::LogDb;
+use re_log_types::{StoreId, StoreKind};
+
+/// Stores many [`LogDb`]s of recordings and blueprints.
+#[derive(Default)]
+pub struct LogDbHub {
```

Comment:
`LogDb` is not an obvious name given the usage of `Store`.  This seems like it should be named something like `StoreHub`?